app/views/map/_leaflet.html.erb

```diff
@@ -32,8 +32,12 @@
     <% end %>
 
     var markers_hash<%= unique_id %> = new Map() ;
-
-    setupLEL(map<%= unique_id %>, markers_hash<%= unique_id %>);
+    var options<%= unique_id %> = {
+        layers: ['PLpeople'],
```

Comment:
This looks good as it's passed into:

https://github.com/publiclab/plots2/blob/ac0421509036dde1d64f458303b53789b9b7ea2b/app/assets/javascripts/leafletHelper.js#L137

And then uses the 'display' and 'include' options in LEL from https://github.com/publiclab/leaflet-environmental-layers#usage

I suppose eventually we should refactor `setupLEL()` to use the same option names as LEL... but this looks great for now! Thanks!